packages/components/app-icon/types/app-icon.d.ts

```diff
@@ -0,0 +1,5 @@
+export declare class TSIcon {
+	size?: string;
```

Comment:
In this case you are right.
In general, I want to convert our constants to enums, so we could use it as types and provide even better type checks, but it will require some additional changes (e.g. export of available enums), so I planned to do it in following PRs.